torch/autograd/gradcheck.py

```diff
@@ -49,6 +49,11 @@ def get_numerical_jacobian(fn, input, target=None, eps=1e-3):
     Note that `target` may not even be part of `input` to `fn`, so please be
     **very careful** in this to not clone `target`.
     """
+
+    is_complex = input[0].dtype.is_complex
+    if is_complex:
+        eps = eps * 1j
```

Comment:
I think you want `eps + eps * 1j` here to match the previous discussions we had.